### ISSUE TITLE
How to train on Custom Data

### ISSUE THREAD

**Issue by ghost** [Trung-Rei/License-plate-recognition-YOLO#3]
Hello I hope you are doing well. Can you please share How to train this model on my own dataset

**Comment by Trung-Rei** [comment]
Hi, thanks for your interest in my project. I trained the LP detection model using YOLOv4 tiny configure on darknet with an annotated dataset. You can try searching Google for some tutorials. Hope this helps. ^^

**Comment by hammad2008** [comment]
How you train. Can you give requirement file? Any notebook file

**Comment by hammad2008** [comment]
Which notebook is used for training the data?

**Comment by Trung-Rei** [comment]
It's the file `lp_detection.ipynb`. Also, you can pass over the part `Pick some car images` since i forgot deleting that.

**Comment by hammad2008** [comment]
The lp_detection.ipynb code will used to train my dataset? The Pascal VOC format is used?

**Comment by Trung-Rei** [comment]
Yes, I used that notebook for training. And the annotations use yolo format.
Here are the necessary files of my training you can refer to: https://drive.google.com/drive/folders/1-F4NY8Y6eZswUeZvBw4UyqvNJSVcBGGi?usp=sharing

**Comment by nguyenvanquan25062001** [comment]
anh có thể cho em xin file data để train cnn nhận diện ký tự với ạ
